library_panel.go

```diff
@@ -101,22 +101,34 @@ func (c *Client) LibraryPanels() ([]LibraryPanel, error) {
 
 // LibraryPanelByUID gets a library panel by UID.
 func (c *Client) LibraryPanelByUID(uid string) (*LibraryPanel, error) {
-	return c.panel(fmt.Sprintf("/api/library-elements/%s", uid))
+	resp := &LibraryPanelCreateResponse{}
+	path := fmt.Sprintf("/api/library-elements/%s", uid)
+
+	err := c.request("GET", path, nil, nil, &resp)
+	if err != nil {
+		return nil, err
+	}
+
+	return &resp.Result, err
```

Comment:
I think here we should return `nil` for error:
```suggestion
	return &resp.Result, nil
```

Of course if the `if` above is false then `err` will effectively be `nil` and the end result is the same, but it's more future-proof to hardcode the returned `nil` value.